### PR TITLE
docs: Optimize installation documentation

### PR DESCRIPTION
**Pull request by Baoyuantop**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

- Related issues
#783

### CODE REVIEW

File: docs/en/latest/deployments/minikube.md [hunk 1]
```diff
@@ -41,7 +40,7 @@ kubectl create ns ingress-apisix
 helm install apisix apisix/apisix \
   --set gateway.type=NodePort \
   --set ingress-controller.enabled=true \
-  --namespace ingress-apisix
+  --namespace ingress-apisix \
```
Comment (tao12345666333) [inline docs/en/latest/deployments/minikube.md:43]: Please add one more `--set ingress-controller.config.apisix.serviceNamespace=ingress-apisix` flag

**Review by tao12345666333 — COMMENTED** [review]
Thanks!

I think it would be better if you can modify the installation steps on other platforms.

File: docs/en/latest/deployments/minikube.md [hunk 2]
```diff
@@ -41,7 +40,7 @@ kubectl create ns ingress-apisix
...
   --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
```
Comment (Baoyuantop) [inline docs/en/latest/deployments/minikube.md:44]: Does it already exist? @tao12345666333

Comment (tao12345666333) [reply docs/en/latest/deployments/minikube.md:44]: yep. it was introduced by https://github.com/apache/apisix-ingress-controller/pull/747  I forgot

**Comment by tao12345666333** [comment]
> I think it would be better if you can modify the installation steps on other platforms.

@Baoyuantop

**Review by tao12345666333 — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
LGTM!

**Comment by tao12345666333** [comment]
Thanks!

**Pull request merged**